src/api/v1/helpers/leaves.helper.js

```diff
@@ -55,15 +55,15 @@ module.exports = {
         /**
          * Find the Team of the Employee
          */
-        const teamWithUser = await TeamAssociationService
-          .findTeamOfAUser({ userId: req.userInfo.id });
+        // const teamWithUser = await TeamAssociationService
+        //   .findTeamOfAUser({ userId: req.userInfo.id });
```

Comment:
Redundant comments

Reply:
@rohit2sharma95 I didn't removed comments, because earlier we were using TeamAssociation for leave, now as we have removed it we don't need that, but I didn't work on Leave in this branch, I thought it will be better to work for leave in a different branch ?

Reply:
Agree :+1: